faqs/different_versions.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Differences between CC, CC:Tweaked, OC, and OC2
+search: differences different
+---
+Over the years, multiple different computer mods have been created. Each of these mods have their own unique features, and cater to different play styles. Below is a brief overview of the differences between the most popular computer mods.
+
+## Brief Descriptions
+
+**CC (ComputerCraft)**
+
+ComputerCraft is the original version of ComputerCraft, developed by Dan200. It's available for Minecraft 1.6.4 up to 1.12.2. This version is missing many of the modern features that CC:Tweaked contains. The computers are programmed using Lua 5.1.
+
+**CC:Tweaked**
+
+CC:Tweaked is an updated version of ComputerCraft, maintained by SquidDev and many members of the community. It adds many new features on top of the original ComputerCraft, including websocket support, new peripherals, and DFPWM audio support. CC:Tweaked operates on the value of backwards compatibility, attempting to ensure that as many existing programs as possible continue to work without modification. Newer versions of the mod also include support for Lua 5.2.
+
+**OC (OpenComputers)**
+
+OpenComputers is a separate mod from ComputerCraft, developed by Asie. It focuses on a more 'realistic' approach to computers within Minecraft, featuring more complex (but limited) hardware components, and its own set of peripherals. OpenComputers is available for Minecraft 1.6.4 up to 1.12.2, and is also programmed using Lua.
```

Comment:
was developed by Sangar, is maintained by Asie

there is a fairly complete (?) fork that ported the mod to 1.16